src/libraries/AssetConfiguration.sol

```diff
@@ -0,0 +1,250 @@
+/**
+    Created by Arcadia Finance
+    https://www.arcadia.finance
+
+    SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity >=0.4.22 <0.9.0;
+
+/**
+ * @title AssetConfiguration library
+ * @author Arcadia Finance
+ * @notice Implements the bitmap logic to handle the asset configurations
+ */
+library AssetConfiguration {
+
+    struct AssetDetailBitmap {
+        // BITMAP
+        //16 bit (0-15)  : Collateral Ratio
+        //16 bit (16-31) : Liquidity Threshold
+        //16 bit (32-47) : Liquidity reward
+        //16 bit (48-63) : Protocol liquidation fee
+        //8  bit (64-71) : Decimals
+        //1  bit (72)    : Asset is active
+        //1  bit (73)    : Asset is frozen
+        //1  bit (74)    : Asset is paused
+        //1  bit (75)    : Asset borrowing is enabled
+        //52 bit (76-127): Unused
+        uint128 data;
+    }
```

Comment:
@Thomas-Smets @simluyt @j-vp Can there be more parameters added here?

Reply:
At least 128 bits left in the same storage slot, so you can fill that up for sure